rospy_azure_iothub/package.xml

```diff
@@ -14,8 +14,8 @@
   <build_depend>rospy</build_depend>
   <build_depend>std_msgs</build_depend>
 
-  <exec_depend>rospy</exec_depend>
   <exec_depend>rospy</exec_depend>
   <exec_depend>std_msgs</exec_depend>
-  <exec_depend>azure-iothub-device-client-pip</exec_depend>
+  <exec_depend>python-azure-iothub-device-client</exec_depend>
```

Comment:
I renamed `azure-iothub-device-client-pip` to `python-azure-iothub-device-client` to align with the naming convention we saw in rosdistro database.